docs/src/rsh/module/index.md

```diff
@@ -1,6 +1,6 @@
 # {#ref-programs-module} Modules
 
-A Reach @{defn("source file")} is a textual file which specifies a Reach module<a class="ui-tooltip" title="a component of a software program containing one or more routines, it can be a separate file from the main program"><span style="cursor: help;">module</span></a>.
+A Reach @{defn("source file")} is a textual file which specifies a Reach @{tooltip(module, a component of a software program containing one or more routines, it can be a separate file from the main program)}.
```

Comment:
Did this compile? I thought you needed to add `"`?

Reply:
Where do I add the quotes? I have put them in a couple different places and none of them compile. I looked at both his example and his code and could not find where to add them. No matter where I put them, I get the error "tooltip is not defined" and I am not sure what that means

Reply:
@{tooltip("text", "hover")}

Did you rebase?

Reply:
Bah, you're right...I rebased my branch where I am developing, but did not rebase where I generate the docs.

Reply:
Added the "" and verified it built and displayed correctly.